polaris-for-vscode/tsconfig.json

```diff
@@ -6,9 +6,13 @@
     "outDir": "out",
```

Comment:
I think it'd be useful to consolidate on a single convention for a compilation output directory. Currently polaris-react uses `build`, polaris-for-vscode uses `out`, and stylelint-polaris doesn't have anything (but it should because d.ts files should be generated for incremental compilation

Reply:
My vote is `dist`. It's used in all the TypeScript docs/examples, default output directory for `webpack` and `rollup`, `parcel` literally calls their option `distDir`, and it's the most common convention I've seen in open source land.